x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/role/GetRolesRequest.java

```diff
@@ -47,9 +44,16 @@ public String[] names() {
         return names;
     }
 
+    public void nativeOnly(boolean nativeOnly) {
+        this.nativeOnly = nativeOnly;
+    }
+
+    public boolean nativeOnly() {
+        return this.nativeOnly;
+    }
+
     @Override
     public void writeTo(StreamOutput out) throws IOException {
-        super.writeTo(out);
```

Comment:
Likewise local-only -- I'm adding a new field so porting this over to avoid adding dead serialization code.